### ISSUE TITLE
`no-duplicates` false positive

### ISSUE THREAD

**Issue by fr33bits** [un-ts/eslint-plugin-import-x#505]
With Svelte imports I noticed the following issue:

```ts
import { onMount, tick } from 'svelte';
import { quintOut } from 'svelte/easing';
import { scale, slide } from 'svelte/transition';
```

This plugin incorrectly "fixes" these imports by joining them all under a `from 'svelte'` import which is incorrect as `quintOut`, `scale`, `slide` cannot be imported from there (as TypeScript warns about).

This did not use to be an issue with `eslint-plugin-import` and is a result of replacing it with this plugin and upgrading to ESLint v10.
